src/baseSwagger.ts

```diff
@@ -259,9 +261,12 @@ function generateEndpointSwaggerSchema(endpoint: IEndpoint, sharedComponents: Co
 					[method]: endpointMethodSwaggerSchema
 				}
 			} catch (err) {
-				// eslint-disable-next-line no-console
-				console.log(`ERROR with method:${methodData.method} ${path}`, err)
-				return {}
+				return {
+					[endpoint.methods[index].method]: {
```

Comment:
@kot-1999 [endpoint.methods[index].method] this is the same as [method] in try case or ?

Reply:
Yes, it is the same

Reply:
@kot-1999 so it whould be better use the same syntax and do not use index at all

Reply:
Than how I will find out in _catch_ block which endpoint is beeing processed at the momment in _try_ one?
It is possible only due to index

Reply:
@kot-1999  [methodData.method]: {}
btw it already was used in console.log(`ERROR with method:${methodData.method} ${path}`, err)